openproblems/tasks/cell_cell_communication/api.py

```diff
@@ -46,16 +46,16 @@ def check_dataset(adata):
 
     resource = ligand_receptor_resource(adata.uns["target_organism"])
     if "receptor" in adata.uns["ccc_target"].columns:
-        assert np.all(
+        assert np.any(
```

Comment:
Shouldn't we be checking that all of the ligands/receptors are valid?

Reply:
This wasn't passing /w all due to the aliases of the ligands - I reuploaded the dataset now /w `ccc_pred` which includes only the overlapping ligands and have hence changed this to all.

Also, wrote a small helper function to allow to check for all ligands/receptors in `check_methods`